packages/grid/x-data-grid-generator/src/renderer/renderEditRateType.tsx

```diff
@@ -1,65 +0,0 @@
-import * as React from 'react';
```

Comment:
I assume it wasn't used?

Reply:
Yes, I was to lazy to migrate unused packages :D